kapitan/inputs/base.py

```diff
@@ -178,3 +184,35 @@ def __enter__(self):
 
     def __exit__(self, *args):
         self.fp.close()
+
+
+### Option 1: Manually encoding the .data key to base64 (postprocessing)
+
+
+def check_data_for_b64(yml_obj):
+    """
+    check for .data in kind: Secret / ConfigMap
+    replace b64 encoded strings
+    """
+    for item in yml_obj:
+        kind = item.get("kind", None)
+        if kind and (kind == "Secret" or kind == "ConfigMap"):
```

Comment:
I think it's cleaner to do instead `if kind in ('Secret, 'ConfigMap'):`

Reply:
You're right. I restructured the code a bit and applied your suggestion